### PR TITLE
Add periodic info logging of scheduled checks

### PR DESCRIPTION
**Pull request by itzg**

### CODE REVIEW

File: poller/scheduler.go [hunk 1]
```diff
@@ -112,13 +113,30 @@ func (s *EleScheduler) ReconcileChecks(cp ChecksPrepared) {
...
 
 func (s *EleScheduler) runReconciler() {
+
+	logTicker := time.NewTicker(checkLoggerDuration)
+	defer logTicker.Stop()
+
 	for {
 		select {
 		case cp := <-s.preparations:
 			s.reconcile(cp)
```
Comment (itzg) [inline poller/scheduler.go]: @rphillips Maybe I should do the same logging after reconcile here?

File: poller/scheduler.go [hunk 2]
```diff
@@ -112,13 +113,30 @@ func (s *EleScheduler) ReconcileChecks(cp ChecksPrepared) {
...
 
 		case <-s.ctx.Done():
 			return
+
+		case <-logTicker.C:
+			if len(s.checks) > 0 {
+				typeCounts := log.Fields{}
+				for _, ch := range s.checks {
```
Comment (goru97) [inline poller/scheduler.go]: Are we running this for loop after every 5 minutes?

File: poller/scheduler.go (hunk 1 above)
Comment (rphillips) [inline poller/scheduler.go]: agreed

File: poller/scheduler.go [hunk 3]
```diff
@@ -254,6 +272,12 @@ func (s *EleScheduler) runCheckTimerLoop(ch check.Check) {
 
 // Execute perform the default CheckExecutor behavior by running the check and sending its results via SendMetrics.
 func (s *EleScheduler) Execute(ch check.Check) {
+	log.WithFields(log.Fields{
+		"id":     ch.GetID(),
+		"type":   ch.GetCheckType(),
+		"period": ch.GetPeriod(),
+	}).Info("Running check")
```
Comment (rphillips) [inline poller/scheduler.go]: perhaps this should still be debug. thoughts?

File: poller/scheduler.go (hunk 2 above)
Comment (rphillips) [inline poller/scheduler.go]: the loop reads from the first available channel, and will block until a channel is ready to read from, thus the timer triggering every 5 minutes

Comment (goru97) [inline poller/scheduler.go]: Now I understood the power of select statement and channel @itzg was talking about.

File: poller/scheduler.go (hunk 3 above)
Comment (itzg) [inline poller/scheduler.go]: Sorry, yep, fixed in next push

**Pull request merged**